source/guides/clear/mixer.rst

```diff
@@ -397,7 +467,7 @@ mix version 10 (from which the image was built), to mix version 20.
 
    .. code-block:: bash
 
-      swupd bundle add editors
+      swupd bundle-add editors
 
 #. Try to start the `joe` editor.  It should not appear because we removed it
```

Comment:
Lines 472-473 should be:

#. Try to start the `joe` editor.

.. code-block:: bash

   joe

It should not appear because we removed it from the original `editors` bundle.